elastic/datadog_checks/elastic/elastic.py

```diff
@@ -496,6 +514,29 @@ def _get_es_version(self, config):
         self.log.debug("Elasticsearch version is %s" % version)
         return version
 
+    def _get_index_metrics(self, config):
+        cat_url = '/_cat/indices?format=json&bytes=b'
+        index_url = urlparse.urljoin(config.url, cat_url)
+        index_resp = self._get_data(index_url, config)
+        index_stats_metrics = self.INDEX_STATS_METRICS
+        for idx in index_resp:
+            tags = config.tags + ['index_name:' + idx['index']]
+            index_data = {
+                'docs_count':         idx['docs.count'],
```

Comment:
Is there a case where a single one of these may not exist? Do you think it'd be good to more safely try to access these in case specific versions of elastic rename or remove those fields?

Reply:
These fields are the output for `_cat/indices` so if the values do not exist, they will be defaulted to 0. The following is an example of the output.
```
health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size
yellow open   twitter  u8FNjxh8Rfy_awN11oDKYQ   1   1       1200            0     88.1kb         88.1kb
green  open   twitter2 nYFWZEO7TUiOjLQXBaYJpA   5   0          0            0       260b           260b
```

Reply:
What I meant for this was actually if a future version of elastic removes one of those fields, or renames `pri` to `prio`, I think we'd get a KeyError here. I think we should make this behavior so that it doesn't report the value and maybe log a warning that we couldn't retrieve it. What do you think?